backend/django_core/apps/pages/views/user.py

```diff
@@ -160,10 +160,19 @@ def register_view(request: HttpRequest) -> HttpResponse:
                     form.fields["username"].widget.attrs["class"] += " focus:border-error"
 
             return render(request, "user/register/_2.html", context={"form": form})
-        else:
+        elif _internal_state_ == 3:
             form_data = request.session["_form_"]
-            print(form_data)
-            return HttpResponse(200)
+            username = form_data["username"][0] if form_data.get("username") else None
+            email = form_data["email"][0] if form_data.get("email") else None
```

Comment:
What in god's name is this?

Reply:
Ehh you dont get it?
its a simple logic, even my doggy can understand :sweat_smile: